### ISSUE TITLE
aria-relevant with multiple values fails

### ISSUE THREAD

**Issue by asparaguscat** [dequelabs/axe-core#157]
Invalid ARIA attribute value: aria-relevant="additions removals"
Invalid ARIA attribute value: aria-relevant="additions text"
Invalid ARIA attribute value: aria-relevant="removals text"

```
<div aria-relevant="additions removals"></div>
```

**Comment by WilcoFiers** [comment]
We have resolved this issue in our internal repo. This will be available in the next major release of axe-core.
